docs/integrations/scheduled-commands.md

```diff
@@ -0,0 +1,60 @@
+---
+id: scheduled-commands
+title: Scheduled Commands
+---
+
+### available from XSOAR version 6.2.0 and up
+
+<img width="533" src="../doc_imgs/integrations/polling-command.png"></img>
+
+It's possible for a command to schedule a future execution for another command with predetermined `next_run` time and `args` as well as `timeout`.
+
+Use cases for using a scheduled commands include:
+1. ***Polling Flow*** - The command cannot return the full result in a single execution (likely because it's waiting for a remote process to finish execution). Scheduled commands enable to set the command to try again later, and return the full result when it can. example use cases: `Sandbox Detonation`, `Autofocus samples search`.
+2. ***Rate Limiting*** - The command cannot perform the required action because the instance has reached its rate limit. Scheduled commands enable to set the command to try again later to check if the rate limit was removed, so the command can be executed.
+3. ***Concurrency*** - The command cannot perform the required action because the instance has reached its concurrence limit. Scheduled commands enable to set the command to try again later (preferably with an exponential backoff).
```

Comment:
```suggestion
3. ***Concurrency*** - The command cannot perform the required action because the instance has reached its concurrent limit. Scheduled commands are enabled to set the command to try again later (preferably with exponential backoff).
```